structuralcodes/codes/ec2_2023/_section5_materials.py

```diff
@@ -64,50 +64,49 @@ def fctm(fck: float) -> float:
 def fctk_5(_fctm: float) -> float:
     """Compute the 5% mean concrete tensile strength fractile.
 
-    EN 1992-1-1:2023, Table 5.1
+    EN 1992-1-1:2023, Table 5.1.
 
     Args:
-        _fctm (float): the mean concrete tensile strength in MPa
+        _fctm (float): The mean concrete tensile strength in MPa.
 
     Returns:
-        float: the 5% mean concrete tensile strength fractile in MPa
+        float: The 5% mean concrete tensile strength fractile in MPa.
     """
     return abs(_fctm) * 0.7
 
 
 def fctk_95(_fctm: float) -> float:
     """Compute the 95% mean concrete tensile strength fractile.
 
-    EN 1992-1-1:2023, Table 5.1
+    EN 1992-1-1:2023, Table 5.1.
 
     Args:
-        _fctm (float): the mean concrete tensile strength in MPa
+        _fctm (float): The mean concrete tensile strength in MPa.
 
     Returns:
-        float: the 5% mean concrete tensile strength fractile in MPa
+        float: The 5% mean concrete tensile strength fractile in MPa.
     """
     return abs(_fctm) * 1.3
 
 
 def Ecm(_fcm: float, kE: float = 9500) -> float:
     """Computes the secant modulus between sigma_c=0 and sigma_c=0.4*fcm.
 
-    EN 1992-1-1:2023, Eq. (5.1)
+    EN 1992-1-1:2023, Eq. (5.1).
 
     Args:
-        _fcm (float): the mean compressibe strength in MPa
+        _fcm (float): The mean compressive strength in MPa.
```

Comment:
Thanks for pointing it out. I agree that the leading underscore is misplaced. To avoid name conflicts we should instead used a trailing underscore. More about this topic [here](https://realpython.com/python-double-underscore/). However, I also agree that it should not be necessary. I will do an attempt to remove as much as possible of the underscores, and whenever needed, replace the leading ones with trailing.

Reply:
Done in dd9bf62